src/System.Windows.Forms/src/System/Windows/Forms/Control.ActiveXImpl.cs

```diff
@@ -1680,7 +1677,7 @@ private class VTABLE
                     /// </summary>
                     public ComConnectionPoint FindConnectionPoint(Type eventInterface)
                     {
-                        FindConnectionPointD findConnectionPoint = (FindConnectionPointD)Marshal.GetDelegateForFunctionPointer(_vtbl.FindConnectionPointPtr, typeof(FindConnectionPointD));
+                        FindConnectionPointD findConnectionPoint = (FindConnectionPointD)Marshal.GetDelegateForFunctionPointer(_vtbl!.FindConnectionPointPtr, typeof(FindConnectionPointD));
```

Comment:
Since `VTABLE` is a struct, it can't be null
```suggestion
                        FindConnectionPointD findConnectionPoint = (FindConnectionPointD)Marshal.GetDelegateForFunctionPointer(_vtbl.FindConnectionPointPtr, typeof(FindConnectionPointD));
```